packages/lesswrong/server/embeddings.ts

```diff
@@ -10,10 +10,15 @@ import { isAnyTest, isE2E } from "../lib/executionEnvironment";
 import { isEAForum } from "../lib/instanceSettings";
 import { addCronJob } from "./cronUtil";
 import { TiktokenModel, encoding_for_model } from "@dqbd/tiktoken";
+import mapValues from "lodash/mapValues";
+import chunk from "lodash/chunk";
+import { EMBEDDINGS_VECTOR_SIZE } from "../lib/collections/postEmbeddings/schema";
 
 export const HAS_EMBEDDINGS_FOR_RECOMMENDATIONS = isEAForum && !isE2E;
 
 export const DEFAULT_EMBEDDINGS_MODEL: TiktokenModel = "text-embedding-ada-002";
+const NEW_EMBEDDINGS_MODEL = "text-embedding-3-large";
```

Comment:
Could we not just add some settings object like:
```
  // For LessWrong
  "embeddings": {
    "tokenizerModel": "text-embedding-ada-002",
    "embeddingsModel": "text-embedding-3-large",
    "maxTokens": 8191,
    "supportsBatchUpdate": true,
  },
  // For EAF
  "embeddings": {
    "tokenizerModel": "text-embedding-ada-002",
    "embeddingsModel": "text-embedding-ada-002",
    "maxTokens": 8191,
    "supportsBatchUpdate": false,
  },
```

I'm concious of it costing a lot of money the last time we generated embeddings for the entire DB, and it'd be nice to not have to do that again right now.

Reply:
Ok, we've forum-gated it now.  (Though admittedly I'm a little confused re: cost, since it'd cost <$100 to do all the LessWrong posts; maybe prices have dropped a lot?)